packages/speech/src/index.js

```diff
@@ -432,11 +435,19 @@ Speech.prototype.createRecognizeStream = function(config) {
     throw new Error('A recognize request requires a configuration object.');
   }
 
-  config = extend(true, {
-    config: {
-      languageCode: 'en-US'
-    }
-  }, config);
+  // As of Speech v1, a language code is required; throw an exception if we
+  // did not receive one.
+  //
+  // This is expected within a nested config, but in the interest of user
+  // sanity, accept it in the outer config object.
+  config.config = config.config || {};
+  if (config.languageCode) {
+    config.config.languageCode = config.languageCode;
+    delete config.languageCode;
+  }
+  if (is.undefined(config.config.languageCode)) {
+    throw new Error('A `languageCode` is required in the config object.');
```

Comment:
The server will return an error if the language code isn't provided.  Therefore it's not necessary to duplicate that logic here.   (Letting the server do error checking ensures consistency when accessing it through the veneer.  Also, if we ever added automatic language detection on the server and relaxed the constraint, we wouldn't need to update all of the client libraries to let users take advantage of the change.)